go/svc/graphql/pkg/subscription/subscription.go

```diff
@@ -0,0 +1,82 @@
+package subscription
+
+import (
+	"fmt"
+
+	"context"
+
+	"github.com/gomodule/redigo/redis"
+	"github.com/google/uuid"
+	"github.com/ispec-inc/monorepo/go/pkg/msgbs"
+	"github.com/ispec-inc/monorepo/go/svc/admin/pkg/redisbs"
+)
+
+var subscribers = make(map[msgbs.Event][]string)
+var message = make(map[msgbs.Event]chan redis.Message)
```

Comment:
Message, likewise, needs a set of behaviors and data structures, so it might be easier to see it as struct!